enacts/flex_fcst/cpt.py

```diff
@@ -126,8 +144,17 @@ def open_mfdataset_nodask(filenames):
     return xr.concat((xr.open_dataset(f) for f in filenames), 'T')
 
 
-def open_var(path, filepattern):
+def open_var(path, filepattern, SL_dense=True):
     filenames = path.glob(filepattern)
     slices = (xr.open_dataset(f) for f in filenames)
     ds = xr.concat(slices, 'T').swap_dims(T='S')
+    if SL_dense:
+        L = (ds["Ti"].dt.month - ds["S"].dt.month).squeeze()
+        L = (L + 6 * (L -np.abs(L)) / L).values
```

Comment:
Wow, creative way of writing `L = L % 12`

Reply:
haha! yes indeed... but that's because it's not modulo. -2 % 12 is -2, not 10. The goal here is not to change L when positive, and add 12 when it is negative.

Reply:
Oh right, `L = (L + 12) % 12`

Reply:
Haha! oh boy... I used that instead of where now.